src/email/email.service.ts

```diff
@@ -0,0 +1,93 @@
+import { MailerService } from '@nestjs-modules/mailer';
+import { Injectable, Logger } from '@nestjs/common';
+import { Cron, CronExpression } from '@nestjs/schedule';
+import fs from 'fs';
+
+@Injectable()
+export class MailService {
+  private logger = new Logger('EmailService');
```

Comment:
클래스의 name 프로퍼티를 인자로 넣는 것이 좀 더 좋을거 같아요 :D

```suggestion
  private logger = new Logger(MailService.name);
```